src/carts/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+services:
+  web:
+    container_name: carts
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      - PORT=9200
+    ports:
+      - 9200:9200
+    volumes:
+      - .:/app-carts
```

Comment:
on the Dockerfile.dev the workdir is /app instead of /app-carts

Reply:
I updated the volume